dev/make-distribution.sh

```diff
@@ -150,7 +156,13 @@ export MAVEN_OPTS="${MAVEN_OPTS:--Xmx2g -XX:MaxPermSize=512M -XX:ReservedCodeCac
 # Store the command as an array because $MVN variable might have spaces in it.
 # Normal quoting tricks don't work.
 # See: http://mywiki.wooledge.org/BashFAQ/050
-BUILD_COMMAND=("$MVN" clean package -DskipTests $@)
+
+# If NUM_THREADS is set we actually want to add the -T in (removed with shift earlier)
+if [ -n "$NUM_THREADS" ]; then
+  MVN_T_OPTION="-T"
+fi
+
+BUILD_COMMAND=("$MVN" $MVN_T_OPTION $NUM_THREADS clean package -DskipTests $@)
```

Comment:
OK I see. Originally I figured, hey, just always set this to `-T 1C` right here. It doesn't hurt, but also this is a rare thing, to prepare releases. I am not sure it's worth any non-trivial complexity here. What else do we want to plumb through then, etc. I'd either not do this, or hard-code the argument.